Developers/Newcomers_Guide.rst

```diff
@@ -6,6 +6,24 @@ Welcome to the Newcomers Guide!
 This is a step-based guide that will help you get your first contribution
 at coala!
 
+There's a few things a contributor needs in order to be considered a full-norm
+coalaian:
+
+  - merge a ``difficulty/newcomer`` Pull Request
+  - review other Pull Requests
+  - merge a ``difficulty/low`` Pull Request
+
+.. note::
+
+    Please do all these three, because if you're not, and you're just solving
+    a newcomer issue, you're harming us and not helping us.
```

Comment:
maybe just `**Do not only fix a newcomer issue!** Supervising newcomers is really a lot of work. We're all volunteers and we can't keep this up if you don't help us in other areas as well!`? It's still demanding but sounds less harsh IMO